example/pubspec.yaml

```diff
@@ -22,14 +22,16 @@ dependencies:
 #  fhir_auth:
 #    path: ../../fhir/fhir_auth
 
+  file: ^6.1.4
+
```

Comment:
This's the dependency of google_fonts, but this needs to be explicitly defined here because encountered a problem when building the project.